frontend/test/metabase/scenarios/filters/filter.cy.spec.js

```diff
@@ -695,7 +673,7 @@ describe("scenarios > question > filter", () => {
     cy.findByText("Tax").should("not.exist");
   });
 
-  it.skip("should work on twice summarized questions (metabase#15620)", () => {
+  it("should work on twice summarized questions (metabase#15620)", () => {
```

Comment:
I fixed this test to work with the bulk filter modal

Reply:
Did you fix the test or the issue?
If https://github.com/metabase/metabase/issues/15620 is now working, we should close it.

Reply:
oh interesting, I think the issue is still there, but the reproduction doesn't actually reproduce the bug accurately. I'll fix the repro

Reply:
What is inaccurate in the repro (apart from the fact that the sidebar doesn't exist anymore)?

Reply:
Currently, the repro correctly applies a filter, and draws a chart (which is what the assertion is checking) but removes one of the summaries in the process.

![Screen Shot 2022-07-21 at 9 59 34 AM](https://user-images.githubusercontent.com/30528226/180260114-f89ab767-5242-4faa-927e-46fa835c6d34.png)